src/bilder/images/consul/deploy.py

```diff
@@ -0,0 +1,68 @@
+import os
+from pathlib import Path
+
+from pyinfra.api import host
+
+from bilder.components.baseline.steps import install_baseline_packages
+from bilder.components.caddy.models import CaddyConfig, CaddyPlugin
+from bilder.components.caddy.steps import caddy_service, configure_caddy, install_caddy
+from bilder.components.hashicorp.consul.models import (
+    Consul,
+    ConsulACL,
+    ConsulACLToken,
+    ConsulConfig,
+    ConsulTelemetry,
+)
+from bilder.components.hashicorp.consul.steps import proxy_consul_dns
+from bilder.components.hashicorp.steps import (
+    configure_hashicorp_product,
+    install_hashicorp_products,
+    register_services,
+)
+from bilder.facts import has_systemd  # noqa: F401
+
+VERSIONS = {  # noqa: WPS407
+    "caddy_route53": "v1.1.1",
+    "consul": os.environ.get("CONSUL_VERSION", "1.10.0"),
+}
+
+install_baseline_packages()
+consul_configuration = {
+    Path("00-default.json"): ConsulConfig(
+        acl=ConsulACL([ConsulACLToken(master="<value>")]),
```

Comment:
Is this needed here? Seems like it needs to bootstrap first, then have ACLTokens added?

Reply:
The master ACL token is used to bootstrap the overall ACL system (which we aren't currently using really, but the master token is still relevant for the server cluster). I've got a branch started for integrating Sops with a Python wrapper to populate secrets like that one at build time.

Reply:
Alternatively, once I get the Vault ASG deployment running we can pull in values like that via the Vault agent and templates.